### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by tertsdiepraam**
Some maintenance changes:
- license headers
- license file on derive crate
- `derive` -> `uutils-args-derive`
- Move crate docs to readme and include them with `include_str!`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,141 @@
 # uutils-args
 
-An experimental derive-based argument parser specifically for uutils
+Argument parsing for the uutils coreutils project.
```
Comment (cakebaker) [inline README.md:3]:
```suggestion
Argument parsing for the [uutils coreutils](https://www.github.com/uutils/coreutils) project.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,141 @@
...
+API and the internal representation of the settings of your app.
+
+For more information on these traits, see their respective documentation:
+
+- [`Arguments`]
+- [`Options`]
+
+Below is a minimal example of a full CLI application using this library.
+
+```
```
Comment (cakebaker) [inline README.md:45]:
```suggestion
```rust
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,139 @@
...
+
+- [hello world](https://github.com/tertsdiepraam/uutils-args/blob/main/examples/hello_world.rs)
+- [arch](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/arch.rs)
+- [b2sum](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/b2sum.rs)
+- [base32](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/base32.rs)
+- [basename](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/basename.rs)
+- [cat](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/cat.rs)
+- [echo](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/echo.rs)
+- [ls](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/ls.rs)
+- [mktemp](https://github.com/tertsdiepraam/uutils-args/blob/main/tests/coreutils/mktemp.rs)
```
Comment (cakebaker) [inline README.md:139]: Is it on purpose that the examples are not in this repo?

Comment (cakebaker) [inline README.md:139]:
```suggestion
- [hello world](/uutils/uutils-args/blob/main/examples/hello_world.rs)
- [arch](/uutils/uutils-args/blob/main/tests/coreutils/arch.rs)
- [b2sum](/uutils/uutils-args/blob/main/tests/coreutils/b2sum.rs)
- [base32](/uutils/uutils-args/blob/main/tests/coreutils/base32.rs)
- [basename](/uutils/uutils-args/blob/main/tests/coreutils/basename.rs)
- [cat](/uutils/uutils-args/blob/main/tests/coreutils/cat.rs)
- [echo](/uutils/uutils-args/blob/main/tests/coreutils/echo.rs)
- [ls](/uutils/uutils-args/blob/main/tests/coreutils/ls.rs)
- [mktemp](/uutils/uutils-args/blob/main/tests/coreutils/mktemp.rs)
```

Comment (tertsdiepraam) [reply README.md]: The links were absolute because they should also work on docs.rs, not just on GitHub. I've fixed all occurrences in a new commit.

**Pull request merged**